WebDevStudios/wd_s
# Feature/#600 separate stylesheets

Closes https://github.com/WebDevStudios/wd_s/issues/600 

### DESCRIPTION

I added the initial steps to convert our styles to `frontend` and `editor` styles. 

**Update Details**
- Change `index.scss` to `style.scss` to take advantage of WP Scripts default enqueues. Note: All files named `style.scss` will generate a singularly compiled `style-index.css` file.
- Create `editor.scss` to generate a generic stylesheet that can be enqueued directly to the editor. Note: As long as main Sass file is not named `style.scss`, it will generate singularly compiled `index.css` file.
- Add `_s_editor_scripts` for enqueuing editor assets such as `/build/index.css`.

### SCREENSHOTS
Frontend Verification
<img width="389" alt="Screen Shot 2021-03-18 at 5 02 31 PM" src="https://user-images.githubusercontent.com/25035900/111698273-04b73500-880d-11eb-83e7-6b0c4b1439c2.png">
<img width="1657" alt="Screen Shot 2021-03-18 at 5 01 40 PM" src="https://user-images.githubusercontent.com/25035900/111698276-054fcb80-880d-11eb-8a88-99cae7c0899f.png">

Editor Verification
<img width="397" alt="Screen Shot 2021-03-18 at 5 01 16 PM" src="https://user-images.githubusercontent.com/25035900/111698278-05e86200-880d-11eb-917d-b27267f2e6e3.png">
<img width="805" alt="Screen Shot 2021-03-18 at 4 59 27 PM" src="https://user-images.githubusercontent.com/25035900/111698279-05e86200-880d-11eb-9197-3f0feda80cab.png">



### OTHER

- [NA] Is this issue accessible? (Section 508/WCAG 2.0AA)
- [NA] Does this pass CBT?

### STEPS TO VERIFY

How do we test this?

1. Checkout and build files.
2. Verify that styles and tailwind styles on the frontend work.
3. Verify that no consoles errors are triggered on the frontend.
4. Verify that styles and tailwind styles in the editor work.
5. Verify that no consoles errors are triggered in the editor.

### DOCUMENTATION

Will this pull request require updating the wd_s [wiki](https://github.com/WebDevStudios/wd_s/wiki)?

This will require further documentation, once we have a finalized solution. I do not have docs at this moment in time.
